qiskit_nature/problems/second_quantization/lattice/models/fermi_hubbard_model.py

```diff
@@ -63,56 +51,30 @@ def uniform_parameters(
         Returns:
             The Fermi-Hubbard model with uniform parameters.
         """
-        graph = lattice.graph
-        for node_a, node_b, _ in graph.weighted_edge_list():
-            if node_a != node_b:
-                graph.update_edge(node_a, node_b, uniform_hopping)
-
-        for node_a in graph.node_indexes():
-            if graph.has_edge(node_a, node_a):
-                graph.update_edge(node_a, node_a, uniform_onsite_potential)
-            else:
-                graph.add_edge(node_a, node_a, uniform_onsite_potential)
-
-        return cls(Lattice(graph), onsite_interaction)
+        return super().uniform_parameters(
+            lattice, uniform_hopping, uniform_onsite_potential, onsite_interaction
+        )
```

Comment:
Actually this is now completely redundant. I don't think there is a need for this method being overwritten in this subclass.